### ISSUE TITLE
Tabs Disappear When navigating back using fragment backstack

### ISSUE THREAD

**Issue by adfleshner** [aurelhubert/ahbottomnavigation#47]
I am using your lib to create a flexible layout for my android application very nice lib by the way. 

The only problem I am having is when I navigate away from the fragmentA with the bottomNavigation to a fragmentB  without bottomNavigation and then come back to the fragmentA with the bottomNavigation all of the tabs are gone. ( I have run the debugger and it goes through the code to recreate the tabs but the tabs do not show up I have even checked with layoutBounds enabled they just have not been created although all of the tab have been added to the bottomNavigation according to the code ) but then as soon as a configuration change happens ( show or hide of the keyboard, Rotation change, ect... ) the tabs get drawn and are there ready to be used again. Any idea why this maybe happening.

**Comment by adfleshner** [comment]
So I did a little trying to figure things out and noticed if I put the addItem function in A handler and runnable they would appear. So whether I am the only person this is happening to or it is a bug that you never encountered this is my workaround to fix the disappearing tabs. I can look at the lib and create a pull request or leave it to you ( either or I have fixed the problem on my end that the app works great now and the tabs no longer disappear when I go to a different fragment and then come back )
 
final Handler handler = new Handler();
        final Runnable r = new Runnable() {
            public void run() {
                bottomNavigation.addItem(item1);
                bottomNavigation.addItem(item2);
                bottomNavigation.addItem(item3);
                bottomNavigation.addItem(item4);
            }
        };
        handler.postDelayed(r, 1);

**Comment by aurelhubert** [comment]
@adfleshner I have tried to test your implementation: ` I navigate away from the fragmentA with the bottomNavigation to a fragmentB without bottomNavigation and then come back to the fragmentA with the bottomNavigation`

I can't reproduce the bug. Can you share more code to explain what you're doing exactly? I think we can maybe find a better fix! :)

**Comment by adfleshner** [comment]
https://github.com/adfleshner/TwoPaneProject this is my feeler project that I am using to create my current project. if you go into the code and remove the handler code around the add items the disappearing bug reproduces on my end. I am using all marshmallow emulators and devices. 
```java
//Change this 
final Handler handler = new Handler();
       final Runnable r = new Runnable() {
            public void run() {
                bottomNavigation.addItem(item1);
                bottomNavigation.addItem(item2);
                bottomNavigation.addItem(item3);
                bottomNavigation.addItem(item4);
            }
       };
        handler.postDelayed(r, 1);
// to this
//final Handler handler = new Handler();
//       final Runnable r = new Runnable() {
//            public void run() {
                bottomNavigation.addItem(item1);
                bottomNavigation.addItem(item2);
                bottomNavigation.addItem(item3);
                bottomNavigation.addItem(item4);
//            }
//       };
//        handler.postDelayed(r, 1);
```
This code is in the LeftFragment.java File at line 86

**Comment by aurelhubert** [comment]
@adfleshner Thanks, I will take a look to your project Monday.

**Comment by adfleshner** [comment]
@aurelhubert sounds good

**Comment by aurelhubert** [comment]
@adfleshner It should be fixed in the version 1.1.1. Your code and this Stack Overflow post help me to understand what was the problem: http://stackoverflow.com/a/5853333.

**Comment by adfleshner** [comment]
Awesome! Thanks all works great now!
